pkg/mmesh/etcdrangewatcher.go

```diff
@@ -30,17 +30,17 @@ import (
 )
 
 type EtcdConfig struct {
-	Endpoints             string `json:"endpoints"`
-	Username              string `json:"userid"`
-	Password              string `json:"password"`
-	RootPrefix            string `json:"root_prefix"`
-	Certificate           string `json:"certificate"`
-	CertificateFile       string `json:"certificate_file"`
-	ClientKey             string `json:"client_key"`
-	ClientKeyFile         string `json:"client_key_file"`
-	ClientCertificate     string `json:"client_certificate"`
-	ClientCertificateFile string `json:"client_certificate_file"`
-	OverrideAuthority     string `json:"override_authority"`
+	Endpoints             string `json:"endpoints,omitempty"`
```

Comment:
Probably best not to have `omitempty` on this one since it's required (all the others are optional).